src/runtime/polyfillsModules.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 Traceur Authors.
```

Comment:
I was trying to follow the pattern set by `src/runtime/runtime-modules.js`.  You make the call, I'm good either way.